### ISSUE TITLE
Homogenization of parameters

### ISSUE THREAD

**Issue by RobbinBouwmeester** [Proteobench/ProteoBench#380]
E.g., MBR --> T and MBR --> True

**Comment by RobbinBouwmeester** [comment]
@Cajac102, do you know if this is fixed?

**Comment by Cajac102** [comment]
Not yet, became a bit worse even (X!Tandem MBR is parsed with single characters "T" and "F")

**Comment by Cajac102** [comment]
I started standardising the parameters with the i2MassChroQ parser. Some notes:

-mass tolerances: either "x ppm" or "x Da"
-for the enzyme field: I used the approach that SDRF uses, namely regular expression instead of enzyme name (trypsin: (?<=[KR])(?!P) for strict, (?<=[KR]) for nonstrict)

Still an issue:
-variable_mods and fixed_mods: These fields are free text, and not homogenised in any way across search engines. We should think about how to parse these parameters, e.g. add modification dicts and/or patterns to the .tomls

**Comment by mlocardpaulet** [comment]
Just putting it here: maybe we do not need to homogenise them all. I mean: what is important is that all the information is there. Although it would be nice, I think that we do not need to make it homogenous. 
I have not looked into the MSAngel and PD parameters yet, but I feel that it will be yet again different...

**Comment by mlocardpaulet** [comment]
Yes, I am not against it at all. But my personal opinion would be to keep it as a pre-processing step. Not to have it as part of ProteoBench modules parsing. 
And so you know and are prepared: I think that WOMBAT and MSAngel and maybe ProlineStudio will not return the same modification format depending on the search engine chosen in the pipeline. But I'd have to check.
